documentation/1.10/content/userguide/tools/create-aux-image.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Create Auxiliary Image"
+date: 2019-02-23
+draft: false
+weight: 2
+description: "The createAuxImage command creates a new container image with WDT models, archives, and variable files."
+---
+
+
+The `createAuxImage` command helps build a container image from a given base OS image. 
```

Comment:
Add sentences:

The main purpose of this command is for creating images that don't require a Java or WebLogic install, but do require having a WDT install plus optional WDT model, archive, or variable files. This is particular useful for supplying [WebLogic Kubernetes Operator Model-in-Image auxiliary images](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/model-in-image/auxiliary-images/).